nx.json

```diff
@@ -6,7 +6,7 @@
     ".eslintrc.json": "*"
   },
   "affected": {
-    "defaultBase": "master"
+    "defaultBase": "main"
```

Comment:
This affects _this_ repo, not projects using nx console. Since this repo still has master for main branch changing this would cause commands ran in _this_ repo to break, without changing the end user experience.

Reply:
Oh I didn't know that 😓 Thanks for the good look @AgentEnder o7